__tests__/app/auth/error/page.test.tsx

```
@@ -0,0 +1,183 @@
+
+const mockUseSearchParams = useSearchParams as jest.MockedFunction<typeof useSearchParams>;
+const mockSignIn = signIn as jest.MockedFunction<typeof signIn>;
+
+describe('AuthError Page', () => {
+  const mockSearchParams = {
+    get: jest.fn(),
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockUseSearchParams.mockReturnValue(mockSearchParams as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)